src/bin/bpf-linker.rs

```diff
@@ -191,51 +245,37 @@ fn main() {
         error("no input files", clap::error::ErrorKind::TooFewValues);
     }
 
-    let env_log_level = match env::var("RUST_LOG") {
-        Ok(s) if !s.is_empty() => match s.parse::<LevelFilter>() {
-            Ok(l) => Some(l),
-            Err(e) => error(
-                &format!("invalid RUST_LOG value: {e}"),
-                clap::error::ErrorKind::InvalidValue,
-            ),
-        },
-        _ => None,
+    // Configure tracing.
```

Comment:
I think this need plumbing for [`OFF`](https://docs.rs/tracing/latest/tracing/level_filters/struct.LevelFilter.html#associatedconstant.OFF) so we can disable the tracing entirely.

Reply:
Well, I decided to go with Tamir's suggestion and remove that enum completely. Now I'm just defining the argument as `Option<tracing::Level>`. If the argument is not provided (default case), logging is off. Do you agree with that solution?